andromedocs-theme/index.mdx

```diff
@@ -0,0 +1,8 @@
+
+# Introduction
```

Comment:
I think it's better to avoid creating files on theme since can be something the user doesn't want to show. What do you think?